libsolidity/interface/CompilerStack.cpp

```diff
@@ -1475,9 +1475,6 @@ void CompilerStack::generateIR(ContractDefinition const& _contract)
 {
 	solAssert(m_stackState >= AnalysisSuccessful, "");
 
-	if (m_experimentalAnalysis)
-		solThrow(CompilerError, "IR codegen after experimental analysis is unsupported.");
-
```

Comment:
How did it end up in this commit?

Reply:
After the rebase you mean?

Reply:
Yes. It's now in my fixup but it looks like it was originally a part of one of the earlier commits.

Reply:
It's from Matheus' cherry picked PR - obviously, no IR codegen in that one, so he added the throw, which then I removed after I rebased this PR with develop (after Matheus' PR was merged).